lab_5_scrapper/scrapper_config.json

```diff
@@ -1,9 +1,13 @@
 {
-    "seed_urls": [],
-    "headers": {},
-    "total_articles_to_find_and_parse": 0,
-    "encoding": "",
-    "timeout": 0,
+    "seed_urls": ["https://www.gazetavechorka.ru/news/proisshestviya"],
+    "headers": {"Accept-Language": "ru,en;q=0.9",
```

Comment:
make proper indents and formatting. example: https://jsonformatter.curiousconcept.com/

Reply:
done